MakeitMVP/CoffeeChat
# Style onboarding page

## Summary
This code styles the onboarding page.

## Updates
Mockup: 
![image](https://github.com/MakeitMVP/CoffeeChat/assets/63386589/91ed3841-e9ba-45c7-bdf0-5d6afdd7630e)


## Testing Instructions
1. In the develop branch, run `git pull` in your terminal to retrieve all updates, then switch to feature branch with `git checkout lc77-fe-styleOnboardingPage`
2. Start the dev environment with `npm start`
3. In your browser, navigate to the /onboarding route to view the build of the  mockup. Note that the final background image and logo will need to be added once created by designers, and the Join Now font should be updated to match global font settings.
